week-01/assessment/capsuleHotel/src/Hotel.java

```diff
@@ -0,0 +1,142 @@
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+public class Hotel {
+
+    public static void main(String[] args) {
+        System.out.println("Welcome to the Capsules Hotel!");
+
+        Scanner console = new Scanner(System.in);
+
+        System.out.print("What is the number of Capsules? ");
+        int capsulesCount = 0;
+        capsulesCount = Integer.parseInt(console.nextLine());
+        System.out.println(capsulesCount);
+        String[] guestList = new String[capsulesCount];
+
+        System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+        int operation = console.nextInt();
+        boolean exitConfirmation = false;
+        while (operation != 4 || !exitConfirmation) {
+            switch (operation) {
+                case 1:
+                    boolean hasSuccessfullyCheckedIn = checkIn(console, guestList);
+                    while (!hasSuccessfullyCheckedIn) {
+                        hasSuccessfullyCheckedIn = checkIn(console, guestList);
+                    }
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+                case 2:
+                    boolean hasSuccessfullyCheckedOut = checkOut(console, guestList);
+                    while (!hasSuccessfullyCheckedOut) {
+                        hasSuccessfullyCheckedOut = checkOut(console, guestList);
+                    }
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+
+                case 3:
+                    view(console, guestList, capsulesCount);
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+
+                case 4:
+                    exitConfirmation = exit(console);
+                    if (exitConfirmation) {
+                        break;
+                    }
+                    System.out.print("Select from the menu: [1. Check In, 2. Check Out, 3. View, and 4. Exit]");
+                    operation = console.nextInt();
+                    break;
+            }
+        }
+
+    }
+
+    //CHECK IN
+    public static boolean checkIn(Scanner console, String[] guestList) {
+        System.out.println("What is your name? ");
+        String name = console.next();
+
+        System.out.println("What is your capsule number? ");
+
+        int capsuleNumber = console.nextInt() - 1;
+        if (guestList[capsuleNumber] == null) {
+            guestList[capsuleNumber] = name;
+            return true;
```

Comment:
as we discussed I would add a success message here so that the user knows the operation was successful 
```suggestion
             System.out.println("Success :)");
             System.out.printf("%s is booked in capsule #%s.%n", name, (capsuleNumber +1));
            return true;
```